src/Marten/Marten.csproj

```diff
@@ -27,9 +31,10 @@
     <EmbeddedResource Include="Schema\SchemaObjects.sql" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Npgsql" Version="4.1.0-ci.1131" />
```

Comment:
now when npgsql https://www.nuget.org/packages/Npgsql/4.0.2 with jsonb fix is released is this pre-release package necessary ?

Reply:
I think so, but I'll double check that

Reply:
@jacobpovar You're right `4.0.2` is enough.

Reply:
great :)